meilisearch/task.py

```diff
@@ -67,6 +68,37 @@ def get_task(config: Config, uid: int) -> dict[str, Any]:
     return http.get(f"{config.paths.task}/{uid}")
 
 
+def cancel_tasks(config: Config, parameters: dict[str, Any] | None = None) -> TaskInfo:
```

Comment:
Same here, `parameters` isn't fully optional, at least one parameter has to be provided.

```suggestion
def cancel_tasks(config: Config, parameters: dict[str, Any]) -> TaskInfo:
```